osu.Game/Skinning/LegacyManiaSkinConfigurationLookup.cs

```diff
@@ -42,6 +42,8 @@ public enum LegacyManiaSkinConfigurationLookups
         HitPosition,
         ScorePosition,
         LightPosition,
+        StagePaddingTop,
+        StagePaddingBottom,
```

Comment:
Yikes...

Reply:
This was the path of least resistance to make this work.

I'm not sure how these will be handled in the future, but I think we'll want to move *all* of this configuration to `SourceSettings` on respective components so they can be edited in the in-game editor. That's going to have to be done as a batch operation anyway, so I don't see adding new entries here as such a bad thing until then.